e2e/timer-reset-mocks/after-reset-all-mocks/timerAndMock.test.js

```diff
@@ -9,17 +9,17 @@ describe('timers', () => {
   it('should work before calling resetAllMocks', () => {
     jest.useFakeTimers();
     const f = jest.fn();
-    setImmediate(() => f());
-    jest.runAllImmediates();
-    expect(f.mock.calls.length).toBe(1);
+    setTimeout(f, 0);
```

Comment:
all of these changes does not strictly have to be here, but it's needed when we actually start using Lolex. Why not land the changes now 🤷‍♂ 

The reason for this change is that `runAllImmediates` is not supported in the Lolex implementation